tests/tests_change_indication_on_repeat_run.yml

```
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+- hosts: all
+  vars:
+    interface: testnic1
+    type: veth
+  name: Test change indication on repeat run
+  tasks:
+    - include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - include_tasks: tasks/assert_device_present.yml
+    - block:
+        - block:
+          - include_role:
```

```suggestion
            - include_role:
```
Please adjust the indentation as reported  by Github actions.

---

Also, yamllint does not like tasks without names. So ensure to add a name for each task for code readability.